components/org.wso2.carbon.identity.oauth.common/src/main/java/org/wso2/carbon/identity/oauth/common/OAuth2ErrorCodes.java

```diff
@@ -35,4 +35,20 @@
     private OAuth2ErrorCodes(){
 
     }
+
+    public class OAuth2SubErrorCodes {
```

Comment:
Shall we have a class level comment explaining what is this?

Reply:
Added with the PR https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1144